app/views/checklist_tests/_checklist_measure.html.erb

```diff
@@ -72,13 +75,38 @@
                           <i class = 'fa fa-fw fa-times text-danger' aria-hidden="true"></i>
                         <% end %>
                         <%= criteria_field.text_field :code, hide_label: false %>
+                          <div id="lookupModal" class="modal fade" role="dialog">
+                            <div class = "modal-dialog" id ="lookupModal-dialog">
+                              <div class="modal-content" id="lookupModal-content">
+                                <div class= "modal-header" id= "lookupModal-header">
+                                  <button type="button" class="close" id="lookupModal-close" data-dismiss="modal">&times;</button>
+                                    <h4 class="modal-title" id="lookupModal-title"> List of codes</h4>
+                                 </div>
+                                    <div class="modal-body" id="lookupModal-body">
+                                      <input type="text" id="lookupFilter" onkeyup="lookupFunction()" placeholder="Filter codes">
```

Comment:
This input is included as a part of the manual entry form. So if you click into the text box and press enter, it submits the form, which is not the behavior we want, especially because people might not realize that the filter automatically applies as you type. I think you would either need to put this into it's own form or do something like jQuery's [`preventDefault()`](event.preventDefault())